src/main/java/com/netgrif/application/engine/pdf/generator/service/PdfDataHelper.java

```diff
@@ -258,6 +284,9 @@ private void refreshGrid(DataGroup dataGroup) {
             Integer cols = dataGroup.getLayout().getCols();
             resource.setFormGridCols(cols == null ? this.originalCols : cols);
             resource.updateProperties();
+        } else {
+            resource.setFormGridCols(this.originalCols);
+            resource.updateProperties();
```

Comment:
`updateProperties` is done in both cases, consider extracting after if/else block

Reply:
Done